mongo/integration/crud_helpers_test.go

```diff
@@ -833,6 +833,17 @@ func executeUpdateOne(mt *mtest.T, sess mongo.Session, args bson.Raw) (*mongo.Up
 			opts = opts.SetUpsert(val.Boolean())
 		case "collation":
 			opts = opts.SetCollation(createCollation(mt, val.Document()))
+		case "hint":
```

Comment:
Given the number of CRUD helpers that allow hints now, do you think it makes sense to add a `createHint` function that does the string/document checking and returns `interface{}`?

Reply:
done